client/src/components/QnA/SearchQuestions.jsx

```diff
@@ -130,7 +130,7 @@ class SearchQuestions extends React.Component {
       .catch((err) => {
         console.log('Error happened while posting a new question', err);
       });
-
+    window.location.reload();
```

Comment:
this is neat. I have an idea of what its doing but I'm curious. Maybe during standup you can expand. Theres a similar bug in componentDidMount in our app.jsx file.

Reply:
So, instead of reloading manually to see the update when the user asks a new question or adds an answer, it will do it programmatically.